### DISCUSSION TITLE
Expect Plugin: Expectation '0' is not supported

### DISCUSSION THREAD

**Discussion by priya-dsp** [artilleryio/artillery#2798] [Artillery]
Hi @hassy , we have used expect plugin with hasProperty for many endpoints and those expectations is getting passed with ok status however, in console it prints: Expect Plugin: Expectation '0' is not supported. What does that mean ?

**Comment by hassy** [comment]
that error originates here: https://github.com/artilleryio/artillery/blob/main/packages/artillery-plugin-expect/index.js#L139

It could be caused if you have an expectation which looks like this:

```yaml
expect:
  # Note the double array here which would lead to the error above
  -
    - hasProperty: "some_property"
```
